drivers/project_scorecard/app/controllers/project_scorecard/warehouse_reports/scorecards_controller.rb

```diff
@@ -93,6 +93,11 @@ def workflow_action
     end
     helper_method :workflow_action
 
+    def apr_url(text, question)
```

Comment:
It might be somewhat dangerous to use `_url` or `_path` since those are built-in for views and controllers.  Maybe `url_for_apr`?

Reply:
oh, and really this is `link_for_apr` if I'm reading that correctly.